spec/cassette/authentication_spec.rb

```diff
@@ -38,21 +40,25 @@
     end
 
     context 'when not cached' do
+      def auth
+        subject
+      end
```

Comment:
I didn't understand the need for this function here... wouldn't a simple call to `authentication` at line 55 match the conditions required by rubocop?